types/chart.js/index.d.ts

```diff
@@ -56,45 +56,62 @@ declare class Chart {
     static Tooltip: Chart.ChartTooltipsStaticConfiguration;
 }
 declare class PluginServiceStatic {
-    register(plugin: PluginServiceRegistrationOptions): void;
-    unregister(plugin: PluginServiceRegistrationOptions): void;
+    register(plugin: PluginServiceGlobalRegistration & PluginServiceRegistrationOptions): void;
+    unregister(plugin: PluginServiceGlobalRegistration & PluginServiceRegistrationOptions): void;
+}
+
+interface PluginServiceGlobalRegistration {
+    id?: string;
```

Comment:
FYI, a global plugin can also be registered without an ID : [example](https://github.com/chartjs/Chart.js/blob/master/samples/advanced/data-labelling.html#L73)